data/lang/lore/en.json

```diff
@@ -0,0 +1,26 @@
+{
+	"" : {
+        "description" : "",
+        "message" : ""
+    },
+    "SOL_MARS_CYDONIA" : {
+        "description" : "",
+        "message" : "Home of the core systems number one gravball team, the Little Green Men."
+    },
+	"SOL_EARTH_LONDON" : {
+        "description" : "",
+        "message" : "Traditional center of civilization. Has recovered quite well after last centurys flooding disasters, although many of the cultural landmarks will never be restored to their former glory."
+    },
+	"SOL_EARTH_MOSCOW" : {
+        "description" : "",
+        "message" : "Moscow Spaceflight museum is a must see! See how it all started, including an actual moon lander recovered from the moon!"
+    },
+	"SOL_EARTH_OLYMPUS" : {
```

Comment:
SOL_MARS_OLYMPUS
:alien:

Reply:
sol_mars_cydonia you mean?

Reply:
No, Cydonia is another station. The text (ski slope...) shows correct on Olympus Mons station but the token name looked a bit off, that's all.

Reply:
I just abbreviated it because its unique enough with the sol_mars

Reply:
>  its unique enough with the sol_mars

Yes, it would have been, but you named it SOL_**EARTH**_OLYMPUS. I don't know if we're talking about the same thing here and I may have misunderstood everything.

Reply:
oopsies. carbon copy much? yes. I though you were talking about cydonia because of the alien emoji

Reply:
Ah, the Little Green Men team! You need to also fix the string in [SCC0000000013-OlympusMons.json](https://github.com/pioneerspacesim/pioneer/blob/6f311b56b42fdaeac94a9d30d34c45a0e39e393b/data/world/stations/00000000/SCC0000000013-OlympusMons.json). ;)

Reply:
thanks,, seems im not a my best today.